README.md

```diff
@@ -4,10 +4,16 @@ At the sametime, it also helps me to know you a little bit more :)
 
 ## Brief self-introduction [max 500 words]
 
+My name is Mario Döbler, I'm a German exchange student at the NCTU and I'm from the University of Stuttgart. I'm an electrical engineering student and I'm specialising in signal processing and machine learning. I wrote my bachelor thesis about the topic "Second order optimization methods for RNN training", so I'm especially looking forward to the part about CNNs.
```

Comment:
"Second order optimization methods for RNN training"
Hope to have a chance to know more about this!